app/controllers/appliances_controller.rb

```diff
@@ -5,20 +5,20 @@ class AppliancesController < ApplicationController
 
   # GET /appliances
   def index
-    @appliances = Appliance.all
+    @appliances = authorize Appliance.all
```

Comment:
Isn't your policy just verifying user can perform new/index/create/... actions?
If so can't this `authorized` be called in a before action?

Reply:
In this case, being Appliances a catalog that only admins can manage, we have to check whether it is an admin or a non-admin user trying to access this section. This makes it impossible for non-admins to go into /appliances.

Reply:
I misread. Hehe.
I added a before_action to enable authorization for each controller that requires it.